plat/qemu/qemu_console.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2015-2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <console.h>
+#include <pl011.h>
+#include <platform_def.h>
+
+static console_pl011_t console;
+static console_pl011_t crash_console;
+
+void qemu_console_init(void)
+{
+#if MULTI_CONSOLE_API
+	console_pl011_register(PLAT_QEMU_BOOT_UART_BASE,
```

Comment:
Static checks fail: 

 [Required] MISRA C-2012 Rule 17.7 violation: Return value of non-void function console_pl011_register() is not used.

Add (void) here should eliminate the fail.